Caps: 8k HEVC enc test cases failed on Gen10- platforms

Caps: 8k HEVC enc test cases failed on Gen10- platforms, take one 8k hevc encode case on gst-vaapi for example,

12964.	
FAIL
cbr.test(bframes=0,bitrate=5000,case=Peru_8K_HDR_7680x4320_100frames,fps=30,gop=30,profile=main,slices=1)   ( full.test.gst-vaapi.encode . hevc )
464 mils 

[2019-11-21 08:05:17.443477] INFO: slash: CALL: gst-launch-1.0 -vf filesrc location=/opt/media/src/assets/otc-media/yuv/Peru_8K_HDR_7680x4320_100frames_I420.yuv num-buffers=50 ! rawvideoparse format=i420 width=7680 height=4320 framerate=30 ! videoconvert ! video/x-raw,format=NV12 ! vaapih265enc rate-control=cbr keyframe-period=30 num-slices=1 max-bframes=0 bitrate=5000 tune=none ! video/x-h265,profile=main ! h265parse ! filesink location=/opt/media/test-results/vaapi-fits/encode/hevc/38dc9244-0c35-11ea-9928-0242ac120002_0/_0.test.gst-vaapi.encode.hevc/cbr/Peru_8K_HDR_7680x4320_100frames-cbr-main-30-30-1-0-5000k-5000k.h265 (pid: 8023)
[2019-11-21 08:05:17.457603] INFO: slash: Setting pipeline to PAUSED ...
[2019-11-21 08:05:17.460851] INFO: slash: Pipeline is PREROLLING ...
[2019-11-21 08:05:17.461029] INFO: slash: Got context from element 'vaapiencodeh265-0': gst.vaapi.Display=context, gst.vaapi.Display=(GstVaapiDisplay)"\(GstVaapiDisplayDRM\)\ vaapidisplaydrm1";
[2019-11-21 08:05:17.600949] INFO: slash: /GstPipeline:pipeline0/GstRawVideoParse:rawvideoparse0.GstPad:src: caps = video/x-raw, format=(string)I420, width=(int)7680, height=(int)4320, interlace-mode=(string)progressive, pixel-aspect-ratio=(fraction)1/1, chroma-site=(string)mpeg2, colorimetry=(string)bt2020, framerate=(fraction)30/1
[2019-11-21 08:05:17.602860] INFO: slash: ERROR: from element /GstPipeline:pipeline0/GstRawVideoParse:rawvideoparse0: Internal data stream error.
[2019-11-21 08:05:17.602947] INFO: slash: Additional debug info:
[2019-11-21 08:05:17.603067] INFO: slash: ../libs/gst/base/gstbaseparse.c(3678): gst_base_parse_loop (): /GstPipeline:pipeline0/GstRawVideoParse:rawvideoparse0:
[2019-11-21 08:05:17.603142] INFO: slash: streaming stopped, reason not-negotiated (-4)
[2019-11-21 08:05:17.603212] INFO: slash: ERROR: pipeline doesn't want to preroll.
[2019-11-21 08:05:17.603275] INFO: slash: Setting pipeline to PAUSED ...
[2019-11-21 08:05:17.603333] INFO: slash: Setting pipeline to READY ...
[2019-11-21 08:05:17.604717] INFO: slash: Setting pipeline to NULL ...
[2019-11-21 08:05:17.604934] INFO: slash: Freeing pipeline ...
[2019-11-21 08:05:17.610077] NOTICE: slash: DETAIL: time(gst:1) = 0.1754s
[2019-11-21 08:05:17.614742] TRACE: slash.core.error: Error added: AssertionError: CALL ERROR: failed with exitcode 1 (pid: 8023)

--- a/lib/caps/APL/iHD
+++ b/lib/caps/APL/iHD
@@ -22,7 +22,7 @@ caps = dict(
   ),
   encode  = dict(
     avc     = dict(maxres = res4k , fmts = ["NV12"]),
-    hevc_8  = dict(maxres = res8k , fmts = ["NV12"]),
+    hevc_8  = dict(maxres = res4k , fmts = ["NV12"]),
   ),
   vdenc   = dict(
     avc     = dict(maxres = res4k , fmts = ["NV12"], cbr = False, vbr = False),

--- a/lib/caps/CFL/iHD
+++ b/lib/caps/CFL/iHD
@@ -26,8 +26,8 @@ caps = dict(
     avc     = dict(maxres = res4k , fmts = ["NV12"]),
     mpeg2   = dict(maxres = res2k , fmts = ["NV12"]),
     vp8     = dict(maxres = res4k , fmts = ["NV12"]),
-    hevc_8  = dict(maxres = res8k , fmts = ["NV12"]),
-    hevc_10 = dict(maxres = res8k , fmts = ["P010"]),
+    hevc_8  = dict(maxres = res4k , fmts = ["NV12"]),
+    hevc_10 = dict(maxres = res4k , fmts = ["P010"]),
   ),
   vdenc   = dict(
     avc     = dict(maxres = res4k , fmts = ["NV12", "YUY2", "YUYV", "YVYU", "UYVY", "AYUV"], cbr = False, vbr = False),

--- a/lib/caps/KBL/iHD
+++ b/lib/caps/KBL/iHD
@@ -27,8 +27,8 @@ caps = dict(
     avc     = dict(maxres = res4k , fmts = ["NV12"]),
     mpeg2   = dict(maxres = res2k , fmts = ["NV12"]),
     vp8     = dict(maxres = res4k , fmts = ["NV12"]),
-    hevc_8  = dict(maxres = res8k , fmts = ["NV12"]),
-    hevc_10 = dict(maxres = res8k , fmts = ["P010"]),
+    hevc_8  = dict(maxres = res4k , fmts = ["NV12"]),
+    hevc_10 = dict(maxres = res4k , fmts = ["P010"]),
   ),
   vdenc   = dict(
     avc     = dict(maxres = res4k , fmts = ["NV12", "YUY2", "YUYV", "YVYU", "UYVY", "AYUV"], cbr = False, vbr = False),

--- a/lib/caps/SKL/iHD
+++ b/lib/caps/SKL/iHD
@@ -21,7 +21,7 @@ caps = dict(
   encode  = dict(
     avc     = dict(maxres = res4k , fmts = ["NV12"]),
     mpeg2   = dict(maxres = res2k , fmts = ["NV12"]),
-    hevc_8  = dict(maxres = res8k , fmts = ["NV12"]),
+    hevc_8  = dict(maxres = res4k , fmts = ["NV12"]),
   ),
   vdenc   = dict(
     avc     = dict(maxres = res4k , fmts = ["NV12"], cbr = False, vbr = False),

--- a/lib/caps/WHL/iHD
+++ b/lib/caps/WHL/iHD
@@ -25,8 +25,8 @@ caps = dict(
     avc     = dict(maxres = res4k , fmts = ["NV12"]),
     mpeg2   = dict(maxres = res2k , fmts = ["NV12"]),
     vp8     = dict(maxres = res4k , fmts = ["NV12"]),
-    hevc_8  = dict(maxres = res8k , fmts = ["NV12"]),
-    hevc_10 = dict(maxres = res8k , fmts = ["P010"]),
+    hevc_8  = dict(maxres = res4k , fmts = ["NV12"]),
+    hevc_10 = dict(maxres = res4k , fmts = ["P010"]),
   ),
   vdenc   = dict(
     avc     = dict(maxres = res4k , fmts = ["NV12", "YUY2", "YUYV", "YVYU", "UYVY", "AYUV"], cbr = False, vbr = False),

